amo2kinto/tests/test_generator.py

```diff
@@ -0,0 +1,250 @@
+    "guid": "sqlmoz@facebook.com",
+    "blockID": "i454",
+    "details": {
+        "who": "All Firefox users who have this extension installed.",
+        "created": "2013-05-13T09:43:07Z",
+        "name": "Mozilla Service Pack (malware)",
+        "bug": "https://bugzilla.mozilla.org/show_bug.cgi?id=871610",
+        "why": "This extension is malware posing as Mozilla software. "
+        "It hijacks Facebook accounts and spams other Facebook users."
+    },
+    "versionRange": [{
+        "targetApplication": [
+            {"guid": constants.FIREFOX_APPID,
+             "minVersion": "3.6",
+             "maxVersion": "3.6.*"},
+            {"guid": "{some-other-application}",
+             "minVersion": "1.2",
+             "maxVersion": "1.4"}
+        ],
+        "minVersion": "0",
+        "maxVersion": "*",
+        "severity": 3
+    }, {
+        "targetApplication": [
+            {"guid": "{some-other-application}",
+             "minVersion": "1.2",
+             "maxVersion": "1.4"}
+        ],
+        "minVersion": "0",
+        "maxVersion": "*",
+        "severity": 3
+    }, {
+        "targetApplication": [],
+        "minVersion": "0",
+        "maxVersion": "*",
+        "severity": 0
+    }],
+    "prefs": ["test.blocklist"]
+}
+
+
+PLUGIN_DATA = {
+    "id": "6a1b6dfe-f463-3061-e8f8-6e896ccf2a8a",
+    "last_modified": 1301581706645,
+    "blockID": "p26",
+    "matchName": "^Yahoo Application State Plugin$",
+    "matchFilename": "npYState.dll",
+    "matchDescription": "^Yahoo Application State Plugin$",
+    "infoURL": "https://get.adobe.com/flashplayer/",
+    "details": {
+        "who": "Users of all versions of Yahoo Application State Plugin "
+        "for Firefox 3 and later.\r\n\r\nUsers of all versions of Yahoo "
+        "Application State Plugin for SeaMonkey 1.0.0.5 and later.",
+        "created": "2011-03-31T16:28:26Z",
+        "name": "Yahoo Application State Plugin",
+        "bug": "https://bugzilla.mozilla.org/show_bug.cgi?id=421993",
+        "why": "This plugin causes a high volume of Firefox and SeaMonkey "
+        "crashes."
+    },
+    "versionRange": [{
+        "minVersion": "0",
+        "maxVersion": "4.1.10328.0",
+        "severity": 0,
+        "vulnerabilityStatus": 1,
+        "targetApplication": [{
+            "minVersion": "3.0a1",
+            "guid": constants.FIREFOX_APPID,
+            "maxVersion": "3.*"
+        }]
+    }]
+}
+
+
+def test_iso2human():
+    assert iso2human(ADDONS_DATA['details']['created']) == 'May 13, 2013'
+    assert iso2human(PLUGIN_DATA['details']['created']) == 'Mar 31, 2011'
+
+
+def test_get_template():
+    assert isinstance(get_template('collection.tpl'), Template)
+    assert isinstance(get_template('record.tpl'), Template)
+
+
+def test_generate_index():
+    records = [ADDONS_DATA, PLUGIN_DATA]
+    template = get_template('collection.tpl')
+    f = mock.mock_open()
+    with mock.patch('amo2kinto.generator.open', f, create=True):
```

Comment:
Is this the same as `with mock.patch('amo2kinto.generator.open', create=True) as f`?